news/2018-09-18-the-qat-gazette-august-2018.md

```diff
@@ -0,0 +1,74 @@
+
+From the Quality Assurance Team, we would also like to recognize the incomparable efforts of [Deif](https://osu.ppy.sh/users/318565) and [p3n](https://osu.ppy.sh/users/123703), who have served admirably in their respective fields for an incredible **four years**.
+
+It is no exaggeration to state that their tireless efforts have helped shape osu!'s mapping and modding communities into the successful force they are today, having assisted and mentored a countless number of mappers along the way. Each will receive a profile badge advertising their length of service which they are encouraged to display with pride!
+
+## Within the Beatmap Nominators
+by [Protastic101](https://osu.ppy.sh/users/6712747)
+
+This month was fairly relaxed, without much movement in or out of the Beatmap Nominators. Applications for the Beatmap Nominators have not yet opened, but there are some exciting new changes to the way applications are organized that will surely be worth the wait. Read on to find out more!
+
+### Status of Beatmap Nominator Applications
+
+Some of you may have noticed that Beatmap Nominator applications were not announced on their usual schedule. For those wondering where applications are, we are in the process of revamping the scoring algorithm, to better account for the recent mass-migration to beatmap discussions, and to create a more transparent and streamlined experience for both applicants and the evaluating team. Therefore, applications will be postponed a little to avoid any possible imbalances in modding score as a result of an untested system.
+
+This gives future applicants an opportunity to increase their modding score in the meantime, to ensure they meet the threshold once applications do come out. Regardless of when applications re-open, consistent effort and contribution will always be highly valued, so make sure to continue modding regularly in the meantime if you have aspirations to join the team!
+
+### Beatmap Nominator Retirements
+
+This month saw the departure of six of our hard-working Beatmap Nominators: some old, some relatively new—[Ascendance](https://osu.ppy.sh/users/2931883), [Cerulean Veyron](https://osu.ppy.sh/users/1886524), [Fii](https://osu.ppy.sh/users/3922569),  [Garalulu](https://osu.ppy.sh/users/757783), [riffy](https://osu.ppy.sh/users/597957) and [Sieg](https://osu.ppy.sh/users/1404615). Whether it’s for university study, military service, or simply a well-needed rest, your contributions have been greatly appreciated and we wish you the best of luck in whichever path you now pursue.
+
+## Within the Quality Assurance Team
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+Most of this last month's news and developments centred the discussion and outcomes of a Quality Assurance Team meeting. Usually we like to hold these publicly, but sometimes time isn't on our side and we really need to discuss things at short notice, so apologies for that! A recap of the key aspects of this meeting can be found below.
```

Comment:
centred -> centered (not that it is wrong it's just that I'd never say centred so it doesn't look like I wrote this)